lib/Runtime/Library/JavascriptExternalFunction.cpp

```diff
@@ -273,7 +273,13 @@ namespace Js
 
         ScriptContext * scriptContext = externalFunction->type->GetScriptContext();
         AnalysisAssert(scriptContext);
-        Var result = NULL;
+        Var result = nullptr;
+
+        StdCallJavascriptMethodInfo info = {
+            callInfo.Count > 0 ? args[0] : scriptContext->GetLibrary()->GetNull(),
+            args.HasNewTarget() ? args.GetNewTarget() : args.IsNewCall() ? function : scriptContext->GetLibrary()->GetNull(),
```

Comment:
As discussed in another comment; perhaps this should default to `undefined` instead of `null`? And is it worth asserting that `args.HasNewTarget()` implies that `args.IsNewCall()`?

Reply:
I'll make the change to undefined rather than null - this is per spec even. I don't think it's worth adding that assert here. It might actually be possible to get `new.target` without being in a new call. Don't think there are any ways for it to happen yet, but spec doesn't forbid that case.